inst/tests/tests.Rraw

```diff
@@ -13626,6 +13633,54 @@ dx = data.table(id = 1L, key = "id")
 di = list(z=c(2L, 1L))
 test(1999.2, key(dx[di]), NULL)
 
+# chmatchdup test from benchmark at the bottom of chmatch.c
+set.seed(45L)
+x = sample(letters, 1e6, TRUE)
+y = sample(letters, 1e7, TRUE)
```

Comment:
76.3 MB mem resource, probably the most mem consuming unit test, still not much

Reply:
Good point. I was only thinking about the time of the test (0.09s) and didn't think about the time or space for the sample. 10x smaller would be better. Will do.